### PR TITLE
Feature/aggregate strategies api

### PR DESCRIPTION
**Pull request by ivarconr**
Adds initial support for (see #102).

In this first implementation unleash-api will return both the new `strategies` property along with the old deprecated `strategy` and `parameters` property. These deprecated props will be removed in the next version of . 

The unleash-api will translate "legacy" format into to the new format when it recieves POST requests to create or PUT to update feature toggles.

### CODE REVIEW

**Comment by ivarconr** [comment]
Before this PR is merged i will perform some testing with actual prod-data to see that it behaves as ut should.

File: packages/unleash-api/lib/db/feature.js [hunk 1]
```diff
@@ -74,6 +70,7 @@ module.exports = function (db, eventStore) {
     }
 
     function updateFeature (data) {
+        console.log(data);
```
Comment (sveisvei) [inline packages/unleash-api/lib/db/feature.js]: remove logger.

File: packages/unleash-api/notes/schema.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+## Table: _features_
+
+| **NAME**       | **TYPE**      | **SIZE**        | **NULLABLE** | **COLUMN_DEF** | **COMMENT**    |
+| -------------  | ---------     | -----------     | ------------ | -------------- | -----------    |
+| created_at     | timestamp     | 29              | 1            | now()          |                |
+| name           | varchar       | 255             | 0            | (null)         |                |
+| enabled        | int4          | 10              | 1            | 0              |                |
+| description    | text          | 2147483647      | 1            | (null)         |                |
+| archived       | int4          | 10              | 1            | 0              |                |
+| parameters     | json          | 2147483647      | 1            | (null)         | deprecated (*) |
```
Comment (sveisvei) [inline packages/unleash-api/notes/schema.md]: Remove these deleted fields.

File: packages/unleash-api/lib/db/feature.js (hunk 1 above)
Comment (ivarconr) [inline packages/unleash-api/lib/db/feature.js]: :+1:

**Comment by ivarconr** [comment]
@sveisvei I want to merge this PR ready as it solves the unleash-api-part of #102. The API should be compatible with the existing admin dashboard (but you won't of course be able to use aggregate strategies before this feature is implemented for the UI)

**Comment by sveisvei** [comment]
wfm

**Pull request merged**